pyproject.toml

```
@@ -130,5 +130,5 @@ omit = [".env/*"]
 sort = "Cover"
 
 [tool.codespell]
-ignore-words-list = "3rt,ans,crate,damon,fo,followings,hist,iff,kwanza,mater,secant,som,sur,tim,zar"
+ignore-words-list = "3rt,ans,crate,damon,fo,followings,hist,iff,kwanza,manuel,mater,secant,som,sur,tim,zar"
```

```suggestion
ignore-words-list = "3rt,ans,crate,damon,fo,followings,hist,iff,kwanza,Manuel,mater,secant,som,sur,tim,zar"
```
Not entirely sure how codespell handles ignored words, but would this help codespell recognize "Manuel" as a name while still catching  "manuel" as a typo?

---

These values MUST all be in lowercase as discussed in the codespell docs.